sass.html.markdown

```diff
@@ -6,6 +6,7 @@ contributors:
   - ["Sean Corrales", "https://github.com/droidenator"]
   - ["Kyle Mendes", "https://github.com/pink401k"]
   - ["Keith Miyake", "https://github.com/kaymmm"]
+  - ["César Suárez", "https://github.com/csuarez"]
```

Comment:
sorry, but I'm afraid we reserve the contributor tag to significant contributions :). LGTM otherwise